src/components/vendor-tree.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import TreeView from 'react-treeview';
+
+import '../styles/react-treeview.css';
+
+const vendorArr = ['AMD', 'Intel', 'IBM', 'Cirix', 'Harris', 'TI'];
```

Comment:
no need to define constants

just create `processors.json` with all processors and get vendors and processors names from there